src/reference/features/logging/search.md

```diff
@@ -9,9 +9,7 @@ This extra inclusion allows you to sort through the results of all your testing
 <img alt="Search overview." src="/_images/search_tab.png" center/>
 <br>
 
-> PRO FEATURE
-
-An HTTPQL query search bar is available within Search for Caido Pro users.
+**Caido Pro Feature**: An HTTPQL query search bar is available within Search for Pro users.
```

Comment:
I am not convinced by this method of emphasis for the the Pro features, maybe we could use a special color for the Pro "marker". Thoughts @Corb3nik